README.md

```diff
@@ -140,22 +140,21 @@ machine. Each process is referenced by the port it is listening on and its
 environment variables. TLS information is provided for a port when the port is
 expecting TLS connections.
 
-### example queries
+### Queries
 
-Finding all of the hosts which are listening on a particular port:
+To analyze the results of the database, you can use the database schema documented
+above to craft your own SQL query, or use some of the example queries stored in the
+`example_queries` directory.
 
-``` sql
-SELECT hosts.NAME
-FROM ports
-  JOIN processes
-    ON ports.process_id = processes.id
-  JOIN hosts
-    ON processes.host_id = hosts.id
-WHERE ports.number = 6061
-  AND upper(ports.state) = "LISTEN"
-```
+Finding all of the hosts which are listening on a particular port: hosts_on_port.sql
```

Comment:
A single newline in Markdown does not actually split it onto separate lines.
( https://gist.github.com/shaunlebron/746476e6e7a4d698b373 )

Reply:
@embidem

Reply:
@embidem was this intended to be multiple lines?
Please advise

Reply:
Resolved in https://github.com/pivotal-cf/scantron/pull/8